openghg/store/base/_base.py

```diff
@@ -204,13 +204,37 @@ def update_metadata(self, data: Dict, input_parameters: dict, additional_metadat
 
         return data
 
-    def get_lookup_keys(self, optional_metadata: Optional[Dict]) -> List[str]:
+    def check_info_keys(self, optional_metadata: Optional[Dict]) -> None:
+        """Check the informational metadata is not being used to set required keys.
+
+        Args:
+            optional_metadata: Additional informational metadata
+
+        Returns:
+            None
+            Raises ValueError if any keys within optional_metadata are
+                within the required set of keys.
```

Comment:
I know we don't have an official docstring format, but I think there are some formats where "Raises" is a new section like "Args" and "Returns".

Also, is `optional_metadata` optional here so that you don't have to check when using the this function?

Reply:
Yeah you're right - [Google style guide](https://google.github.io/styleguide/pyguide.html) gives:
```
Raises:
      ConnectionError: If no available port is found.
```